src/data/pyproject.toml

```diff
@@ -24,6 +19,8 @@ dependencies = [
     "datasketch<2.0.0",
     "nlpo3 >=1.3.0, <2",
     "beautifulsoup4==4.12.2, < 5",
-    "scikit-learn==1.2.2, < 2"
+    "scikit-learn==1.2.2, < 2",
+    "PyPDF2",
+    "pdf_table2json",
+    "difflib",
```

Comment:
We don’t need this line. Difflib is already in python default library https://docs.python.org/3/library/difflib.html